packages/divider/src/assets/dividerSVG.tsx

```diff
@@ -0,0 +1,32 @@
+export const dividerSVG = (inverted: boolean): React.ReactElement => {
```

Comment:
I would remove this file and consolidate the logic in `Divider`

Reply:
Moved logic into `Divider.tsx`